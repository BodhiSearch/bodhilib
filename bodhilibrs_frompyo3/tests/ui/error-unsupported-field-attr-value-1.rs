use frompyo3::FromPyO3;

#[derive(FromPyO3)]
struct TestUnsupportedFieldAttrMultiple {
  #[frompyo3(dict = metadata)]
  path: String,
}

fn main() {}
