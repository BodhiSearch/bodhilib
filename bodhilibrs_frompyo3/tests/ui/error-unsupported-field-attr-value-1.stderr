error: Unsupported field attribute value format
 --> tests/ui/error-unsupported-field-attr-value-1.rs:5:21
  |
5 |   #[frompyo3(dict = metadata)]
  |                     ^^^^^^^^
